packages/editor/src/client/global.less

```diff
@@ -1,9 +1,9 @@
-.highlighted-row,
-.highlighted-row .rs-table-cell-group .rs-table-cell {
-  background-color: #f2faff;
-}
-
 .rs-table {
+  .highlighted-row,
+  .highlighted-row &-cell-group &-cell {
```

Comment:
Out of curiosity, what does &-cell-group means here? same as `.rs-table-cell-group` ?

Reply:
@bishoymelek yes